tests/drivers/watchdog/wdt_error_cases/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&wdt010 {
+   status = "okay";
+};
+
+&wdt011 {
+   status = "disabled";
+};
```

Comment:
Disabling nodes disabled by default is redundant.

Reply:
Idea behind this is to have a list of WDT instances that can be easily "switched" ON/OFF.